spec/runtime/with_clean_env_spec.rb

```diff
@@ -3,8 +3,9 @@
 RSpec.describe "Bundler.with_env helpers" do
   describe "Bundler.original_env" do
     before do
+      bundle "config path vendor/bundle"
       gemfile ""
-      bundle "install --path vendor/bundle"
+      bundle "install"
     end
 
     it "should return the PATH present before bundle was activated" do
```

Comment:
@segiddins so this is breaking on travis because it's ships a version of ruby 1.9 that sits in `/usr/bin`. And the `bundle!` is running on that install of ruby. Do you have a quick thoughts on how to address this?